tap_wikipedia/models/wiki_subdirectory.py

```diff
@@ -0,0 +1 @@
+WIKI_SUBDIRECTORY = "/wiki"
```

Comment:
Doesn't feel like a model to me. Put in a module `tap_wikipedia.constants`?

Reply:
I have moved this, and similar, to a `constants` module.